src/remoteConnector.ts

```diff
@@ -760,12 +345,12 @@ export class RemoteConnector extends Disposable {
 							this.logService.error('No SSH gateway:', e);
 							const ok = 'OK';
 							await this.notificationService.showWarningMessage(`${e.host} does not support [direct SSH access](https://github.com/gitpod-io/gitpod/blob/main/install/installer/docs/workspace-ssh-access.md), connecting via the deprecated SSH tunnel over WebSocket.`, { flow: gatewayFlow, id: 'no_ssh_gateway' }, ok);
-							// Do nothing and continue execution
+							return undefined;
 						} else if (e instanceof SSHError && e.message === 'Timed out while waiting for handshake') {
 							this.logService.error('SSH test connection error:', e);
 							const ok = 'OK';
 							await this.notificationService.showWarningMessage(`Timed out while waiting for the SSH handshake. It's possible, that SSH connections on port 22 are blocked, or your network is too slow. Connecting via the deprecated SSH tunnel over WebSocket instead.`, { flow: gatewayFlow, id: 'ssh_timeout' }, ok);
```

Comment:
we should remove here as well `Connecting via the deprecated SSH tunnel over WebSocket instead` and point to troubleshooting docs too

Reply:
we either just skip it or use https://github.com/gitpod-io/gitpod-vscode-desktop/pull/97/files#r1346338466